companies/cdprojekt.json

```diff
@@ -0,0 +1,25 @@
+{
+    "slug": "cdprojekt",
+    "relevant-countries": [
+        "all"
+    ],
+    "name": "CD PROJEKT S.A.",
+    "runs": [
+        "GOG"
+    ],
+    "address": "ul. Jagiellońska 74\nbuilding E\n03-301 Warschau\nPolen",
```

Comment:
When the company is relevant to non-German countries (as is the case here), we usually have the address in English.